core/compiler.common.jvm/src/org/jetbrains/kotlin/load/java/JavaNullabilityAnnotationSettings.kt

```diff
@@ -58,6 +58,11 @@ val NULLABILITY_ANNOTATION_SETTINGS: NullabilityAnnotationStates<JavaNullability
             sinceVersion = KotlinVersion(1, 8),
             reportLevelAfter = ReportLevel.STRICT
         ),
+        FqName("jakarta.annotation") to JavaNullabilityAnnotationsStatus(
+            reportLevelBefore = ReportLevel.WARN,
+            sinceVersion = KotlinVersion(2, 1),
```

Comment:
I'm not sure we can enable STRICT level for the already released version, probably it should be 2.3 instead.
@serras WDYT?

Reply:
I don't think it either, but the final call is done by @zarechenskiy